lib/google/apis/generator/templates/_method.tmpl

```diff
@@ -15,7 +15,7 @@
 #   <%=  block_comment(param.description, 0, 3) %>
 <%   end -%>
 <% end -%>
-<% for param in api.parameters.values.reject {|p| p.name == 'key'} -%>
+<% for param in api.parameters.values.sort_by {|p| p.name}.reject {|p| p.name == 'key'} -%>
```

Comment:
There's a line in annotator.rb that does the blacklisting:

            @rest_description.parameters.reject! { |k, _v| PARAMETER_BLACKLIST.include?(k) }

Returning a new hash each time renders that line ineffective. You can go ahead and move that elsewhere though so you can do the sorting in the model instead of the templates.